lineman/bower.json

```diff
@@ -10,7 +10,7 @@
     "faye": "latest",
     "angular-translate": "~2.7.2",
     "angular-translate-loader-url": "~2.7.2",
-    "angular-bootstrap": "0.11.2",
+    "angular-bootstrap": "0.13.2",
```

Comment:
Ah ok. I think quite a few of em are worked out at least once with the integration tests, yay!